### PR TITLE
[terra-avatar] Add Accessibility Guide

### PR DESCRIPTION
**Pull request by cschetan77**
### Summary

Adding accessibility guide for terra-avatar.

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-core-docs/src/terra-dev-site/doc/avatar/AccessibilityGuide.6.doc.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <A11yExampleAvatar />
+</div>
+
+```jsx
+const cx = classNames.bind(styles);
+
+const AvatarExample = () => (
+    <Card classNames={cx('card')}>
+        <Card.Body>
+            <Avatar image={exampleAvatarImage} initials="JS" alt="Deep Space" size="1.5em" />
```
Comment (supreethmr) [inline packages/terra-core-docs/src/terra-dev-site/doc/avatar/AccessibilityGuide.6.doc.mdx:35]:
```suggestion
            <Avatar image={exampleAvatarImage} initials="JS" alt="Avatar" size="1.5em" />
```
Can you change avatar image to this image https://github.com/cerner/terra-core/blob/main/packages/terra-core-docs/src/terra-dev-site/doc/demographics-banner/assets/demographicsCat.jpg 

Can we also have an example created for decorative Avatar with empty alt prop ..??

**Comment by cschetan77** [comment]
I'm not sure why these CI and CD pipelines are failing, haven't changed any other files apart from these.

File: packages/terra-core-docs/src/terra-dev-site/doc/avatar/AccessibilityGuide.6.doc.mdx [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import { Badge } from 'terra-avatar/package.json?dev-site-package';
+import A11yExampleAvatar from './example/Avatar/A11yExampleAvatar';
+import A11yExampleDecorativeAvatar from './example/Avatar/A11yExampleDecorativeAvatar';
```
Comment (supreethmr) [inline packages/terra-core-docs/src/terra-dev-site/doc/avatar/AccessibilityGuide.6.doc.mdx:3]:
```suggestion
import A11yExampleAvatar from './example/avatar/A11yExampleAvatar.jsx';
import A11yExampleDecorativeAvatar from './example/avatar/A11yExampleDecorativeAvatar.jsx';
```

Comment (cschetan77) [reply packages/terra-core-docs/src/terra-dev-site/doc/avatar/AccessibilityGuide.6.doc.mdx]: Updated.

**Pull request merged**